docs/deployment/landing_page.rst

```diff
@@ -384,3 +384,27 @@ SecureDrop submissions. We control the servers that store your submissions,
 so no third party has direct access to the metadata or content of what you send us.
 
 Do not discuss leaking or whistleblowing, even with trusted contacts.
+
+How to test your *Landing Page* using Tor Browser
```

Comment:
I agree with you that this whole section could use an information architectural overhaul; as it is, I would prefer to place this new section before the sample landing page content. It is a consideration that should be kept in mind during development and deployment of the landing page, anyway, as it often will require specific configuration changes to the CDN or the webserver to prevent Tor-specific issues.

Reply:
Yep, that makes sense. I moved it just above the content recommendations.